### ISSUE TITLE
Compiled Java plugins do not work

### ISSUE THREAD

**Issue by Lucaskyy** [Konloch/bytecode-viewer#405]
Gradle compiled plugin results in:
```
java.lang.IllegalArgumentException: MALFORMED
        at java.util.zip.ZipCoder.toString(ZipCoder.java:58)
        at java.util.zip.ZipInputStream.readLOC(ZipInputStream.java:300)
        at java.util.zip.ZipInputStream.getNextEntry(ZipInputStream.java:122)
        at the.bytecode.club.bytecodeviewer.plugin.strategies.CompiledJavaPluginLaunchStrategy.loadData(CompiledJavaPluginLaunchStrategy.java:83)
```

Using any archiving program to add classes into ZIP/JAR:
same result as Gradle

Using `javac` and `jar` to compile `Template.java` to a JAR:
```
java.io.EOFException
	at java.util.zip.ZipInputStream.readFully(ZipInputStream.java:405)
	at java.util.zip.ZipInputStream.readLOC(ZipInputStream.java:296)
	at java.util.zip.ZipInputStream.getNextEntry(ZipInputStream.java:122)
	at the.bytecode.club.bytecodeviewer.plugin.strategies.CompiledJavaPluginLaunchStrategy.loadData(CompiledJavaPluginLaunchStrategy.java:83)
```

Loading the plain `Template.java` file:
Works! (different loader strategy: `JavaPluginLaunchStrategy`)

TLDR: `CompiledJavaPluginLaunchStrategy` is broken.

**Comment by Lucaskyy** [comment]
One more solution is to convert the bytes to base64 and then set the base64 string to the textarea.
